### PR TITLE
Enable github login

### PR DESCRIPTION
**Pull request by richardartoul**

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def side_effect(*args):
             return vals[args]
 
         with patch('hyperdash_cli.cli.get_input', Mock(side_effect=side_effect)), patch('sys.stdout', new=StringIO()) as fake_out:
-            hyperdash_cli.login()
+            hyperdash_cli.login(argparse.Namespace(email=True))
 
         expected_output = [
             "Successfully logged in!",
```
Comment (andrewschreiber) [inline tests/test_cli.py]: Can we test Github auth?

File: hyperdash_cli/cli.py [hunk 2]
```diff
@@ -261,12 +269,22 @@ def _is_port_open(host, port):
...
+        password = get_input("Password: ", True)
+        success, default_api_key = _login(email, password)
+        if success:
+            print("Successfully logged in! We also installed: {} as your default API key".format(
+                default_api_key))
+        return
+    
+    if args.github:
+        github(args)
+        return
```
Comment (andrewschreiber) [inline hyperdash_cli/cli.py]: Any reason to style the control flow this way as opposed to `if/elif/else`?

File: hyperdash_cli/constants.py [hunk 3]
```diff
@@ -3,9 +3,7 @@
 import six
 
 
-GITHUB_CLIENT_ID = "4793c654216adde7a478"
```
Comment (andrewschreiber) [inline hyperdash_cli/constants.py]: Have we discarded this ID?

File: tests/test_cli.py (hunk 1 above)
Comment (richardartoul) [reply tests/test_cli.py]: its already tested separately

File: hyperdash_cli/constants.py (hunk 3 above)
Comment (richardartoul) [reply hyperdash_cli/constants.py]: its managed by the server now

File: hyperdash_cli/cli.py (hunk 2 above)
Comment (richardartoul) [reply hyperdash_cli/cli.py]: generally try to avoid unnecessary else's

File: hyperdash_cli/constants.py (hunk 3 above)
Comment (andrewschreiber) [reply hyperdash_cli/constants.py]: Yeah, but is it still a usable ID?

Comment (richardartoul) [reply hyperdash_cli/constants.py]: oh yeah it still works, I'll delete it when spencer updates the prod config

**Pull request merged**